requirements.txt

```diff
@@ -13,3 +13,4 @@ GitPython>=2.0.8
 packaging
 urllib3
 dataclasses>=0.6; python_version<"3.7"
+psutil<6.0.0
```

Comment:
Dlaczego taka wersja?

Reply:
5.8.0 to najnowsza a raczej chcemy przed breaking changes. Widywałem to dosyć często i raczej to jest rozsądne aby dawać górny limit

Reply:
@pitercl what do you think?